src/cli/markdown.rs

```diff
@@ -1,20 +1,17 @@
 // Write Markdown to the terminal
-
-use std::io;
-
 use pulldown_cmark::{Event, Tag};
 
-use super::term2::{color, Attr, Terminal};
+use super::term::{Attr, Color, Terminal};
 
 // Handles the wrapping of text written to the console
-struct LineWrapper<'a, T: Terminal> {
+struct LineWrapper<'a> {
```

Comment:
What drives the change to `dyn Terminal` rather than a type parameter?

Reply:
Because we return a Box<dyn Terminal> from the `term::stdout`.

Reply:
Ok, but why do we do that?